### DISCUSSION TITLE
Utilizzo pinza amperometrica (PZEM 016)

### DISCUSSION THREAD

**Discussion by array81** [indomus/forum#1797] [Hardware & Firmware]
Ho un PZEM 016 modificato con un Wemos per connetterlo a Home Assistant.

Se faccio passare 2 fasi nella pinza posso aspettarmi di avere dal dispositivo la "somma" dei consumi? Mi spiego vorrei misurare i consumi del mio impianto di condizionamento. Ho 2 macchine esterne alimentate da 2 linee distinte. Nel quadro dell'impianto di condizionamento quindi ho 2 interruttore e 2 fasi (sia in ingresso che in uscita). Il quadro è piccolo quindi non riesco ad inserire all'interno 2 PZEM inoltre a me interessa il consumo totale non della singola macchina. Se inserisco nella pinza entrambe le fasi in uscita dagli interruttori il risultato sarebbero lo stesso che misurassi un cavo su cui transita la somma delle 2 correnti dei singoli cavi?
O in alternativa devo usare 2 pinze ed effettuare misurazioni distinte? In tal caso credo dovrò passare a un Shelly EM che ha 2 ingressi.

**Comment by jraziel09** [comment]
Puoi inserire i due conduttori nella pinza, verifica verso e portata della pizza.
